modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -3700,10 +3700,18 @@ export const console: ConsoleNS = {
                             heading: "සමූහය",
                             hint: "සම්බන්ධතාවයේ ඇති කණ්ඩායම් හඳුනා ගන්නා ගුණාංගය නියම කරයි.",
                             label: "කණ්ඩායම් ගුණාංගය",
-                            message: "ඔබ අභිරුචි ගුණාංග සිතියම්ගත කිරීමක් එක් කර " +
-                                "නොමැති බැවින් <1>{{ attribute }}</1> ගුණාංගය පෙරනිමියෙන් <1>කණ්ඩායම් ගුණාංගය</1> " +
-                                "ලෙස සලකනු ඇති බව කරුණාවෙන් සලකන්න.",
+                            mappedRolesAbsentMessage: "ඔබගේ වත්මන් වින්යාසය සමඟ, <1> කණ්ඩායම් ආරෝපණය </ 1> වින්යාස කර නොමැත. " +
```

Comment:
Also no need to care about the max-line length now since it'll interfere with translation IDE plugins.
Can fix these in a followup PR.
```suggestion
                            mappedRolesAbsentMessage: "ඔබගේ වත්මන් වින්යාසය සමඟ, <1> කණ්ඩායම් ආරෝපණය </1> වින්යාස කර නොමැත. " +
```